From 7da2351a4ad387c4c29a13dcb1001986fdfee829 Mon Sep 17 00:00:00 2001
From: Alfredo Moralejo <amoralej@redhat.com>
Date: Tue, 30 Jan 2018 17:44:37 +0000
Subject: [PATCH] Revert "Add tests for empty association set comparison"

A bug in python version in CentOS makes these tests to fail
so let's remove then until fixed.

This reverts commit 0612829fb04565c00f8993b978697a8701204977.
---
 test/ext/test_associationproxy.py | 26 +-------------------------
 1 file changed, 1 insertion(+), 25 deletions(-)

diff --git a/test/ext/test_associationproxy.py b/test/ext/test_associationproxy.py
index d8fd4dc..408c885 100644
--- a/test/ext/test_associationproxy.py
+++ b/test/ext/test_associationproxy.py
@@ -601,30 +601,6 @@ class SetTest(_CollectionOperations):
             self.assert_((p1.children > other) == (control > other))
             self.assert_((p1.children >= other) == (control >= other))
 
-    def test_set_comparison_empty_to_empty(self):
-        # test issue #3265 which appears to be python 2.6 specific
-        Parent = self.Parent
-
-        p1 = Parent('P1')
-        p1.children = []
-
-        p2 = Parent('P2')
-        p2.children = []
-
-        set_0 = set()
-        set_a = p1.children
-        set_b = p2.children
-
-        is_(set_a == set_a, True)
-        is_(set_a == set_b, True)
-        is_(set_a == set_0, True)
-        is_(set_0 == set_a, True)
-
-        is_(set_a != set_a, False)
-        is_(set_a != set_b, False)
-        is_(set_a != set_0, False)
-        is_(set_0 != set_a, False)
-
     def test_set_mutation(self):
         Parent, Child = self.Parent, self.Child
 
-- 
1.8.3.1

